cmd/moby/build.go

```diff
@@ -56,7 +56,7 @@ func getDiskSizeMB(s string) (int, error) {
 
 // Process the build arguments and execute build
 func build(args []string) {
-	var buildOut outputList
+	var buildFormats outputList
```

Comment:
should this PR also `s/outputList/formatList`?

Reply:
fixed